.github/workflows/provision_ec2_tag.yml

```diff
@@ -58,17 +58,17 @@ jobs:
           cloud_watch_logs_stream_name: ecs/test-otel-releases
           aws_vpc_subnet: ${{ secrets.AWS_VPC_SUBNET }}
           repo_name: "newrelic/opentelemetry-collector-releases"
-          ref: "terraform_ec2"
+          ref: "terraform_ec2_move_ci"
 
-      # - name: Destroy infra
-      #   uses: newrelic/fargate-runner-action@main
-      #   with:
-      #     aws_region: us-east-2
-      #     container_make_target: "terraform/destroy-ec2 TAG_OR_UNIQUE_NAME=${{ env.TAG_OR_UNIQUE_NAME }}"
-      #     ecs_cluster_name: caos_otel_releases
-      #     task_definition_name: otel-releases-td
-      #     cloud_watch_logs_group_name: /ecs/test-prerelease-otel-releases
-      #     cloud_watch_logs_stream_name: ecs/test-otel-releases
-      #     aws_vpc_subnet: ${{ secrets.AWS_VPC_SUBNET }}
-      #     repo_name: "newrelic/opentelemetry-collector-releases"
-      #     ref: "terraform_ec2"
+      - name: Destroy infra
+        uses: newrelic/fargate-runner-action@main
+        with:
+          aws_region: us-east-2
+          container_make_target: "test/provision TAG_OR_UNIQUE_NAME=${{ env.TAG_OR_UNIQUE_NAME }}"
```

Comment:
should not be `test/destroy`?

Reply:
Nice catch! Fixed in https://github.com/newrelic/opentelemetry-collector-releases/pull/22/commits/c30aaa77f7135a90afb0dff6961fb3cbe005cf6f